analysis_driver/reader/sample_sheet.py

```diff
@@ -1,180 +0,0 @@
-import csv
```

Comment:
We don't need this anymore, can reinstate it if we want to be able to process runs with handwritten sample sheets while the Lims is down.